### PR TITLE
Update document for latest release

### PR DESCRIPTION
**Pull request by Sahiba-Gupta**
This PR will update the document for 1.4 release.

| GitHub Issue # |
https://github.com/dell/csm/issues/350

### CODE REVIEW

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. No edits are required.

File: content/docs/resiliency/release/_index.md [hunk 1]
```diff
@@ -6,16 +6,13 @@ Description: >
   Dell Container Storage Modules (CSM) release notes for resiliency
 ---
 
-## Release Notes - CSM Resiliency 1.2.0
+## Release Notes - CSM Resiliency 1.4.0
```
Comment (sharmilarama) [inline content/docs/resiliency/release/_index.md:9]: Resiliency release is 1.3.0

Comment (Sahiba-Gupta) [reply content/docs/resiliency/release/_index.md:9]: ok ! Will update it

**Review by hoppea2 — APPROVED** [review]
LGTM

**Review by shanmydell — CHANGES_REQUESTED** [review]
U4V is updated to 10.0 after PowerMax 10 OS release

**Review by shanmydell — APPROVED** [review]

**Review by atye — APPROVED** [review]

File: content/docs/authorization/release/_index.md [hunk 2]
```diff
@@ -6,19 +6,21 @@ Description: >
...
 ---
 
-## Release Notes - CSM Authorization 1.3.0
+## Release Notes - CSM Authorization 1.4.0
 
 ### New Features/Changes
 
-- [CSM-Authorization can deployed with helm](https://github.com/dell/csm/issues/261)
+- CSM 1.4 Release specific changes. ([#350](https://github.com/dell/csm/issues/350))
+- Implementation: enable authorization for csm observability powerscale. ([#413](https://github.com/dell/csm/issues/413))
```
Comment (shaynafinocchiaro) [inline content/docs/authorization/release/_index.md:14]: This is an observability related release note. No changes were made to Authorization in this feature.

Comment (Sahiba-Gupta) [reply content/docs/authorization/release/_index.md:14]: oh okay ! Will create another PR to make this change .

**Pull request merged**